components/org.wso2.carbon.identity.oauth.endpoint/pom.xml

```diff
@@ -159,7 +159,10 @@
         <dependency>
             <groupId>org.wso2.orbit.org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <scope>provided</scope>
+        </dependency>
+        <dependency>
```

Comment:
Shall we add a comment saying that we need to explicitly pack these two jars inside webapp itself. Otherwise this may be removed by someone accidently

Reply:
Done.